Framework/MASPreferencesWindowController.m

```diff
@@ -176,10 +174,7 @@ - (NSToolbarItem *)toolbar:(NSToolbar * __unused)toolbar itemForItemIdentifier:(
         toolbarItem.target = self;
         toolbarItem.action = @selector(toolbarItemDidClick:);
     }
-#if !__has_feature(objc_arc)
-    [toolbarItem autorelease];
-#endif
-    return toolbarItem;
+    return AUTORELEASEOBJ(toolbarItem);
```

Comment:
I know this was already like this in the original code, but I think we should call `AUTORELEASEOBJ` in the `alloc/initWithItemIdentifier:` line.